packages/core/src/renderers/UnknownRenderer.tsx

```diff
@@ -1,8 +1,20 @@
 import * as React from 'react';
-import {Component} from "react";
+import { Component } from 'react';
 
-export class UnknownRenderer extends Component<any, any> {
+/**
+ * Props of an {@link UnknownRenderer}
+ */
+export interface UnknownRendererProps {
+  /**
+   * The type for which no renderer has been found.
+   */
+  type: 'renderer' | 'field';
```

Comment:
do we use this prop anywhere?

Reply:
Yes, check `DispatchField` and `DispatchRenderer`.